src/components/Main/Login/index.js

```diff
@@ -21,24 +24,37 @@ const Login = props => {
   const handleLogin = e => {
     e.preventDefault();
     const { email, password } = loginState;
+    notificationConfig.type = 'warning';
     if (!email || !password) {
-      alert('Пожалуйста, заполните все поля в форме');
+      notificationConfig.message = 'Пожалуйста, заполните все поля в форме';
+      notificationsStore.addNotification(notificationConfig);
     } else {
-      const user = localStorage.getItem(email);
-      if (user) {
-        if (JSON.parse(user).password === password) {
-          console.log();
-          const auth = { isAuthenticated: true };
-          props.handleAuth(auth);
-          localStorage.setItem('auth', JSON.stringify(auth));
+      const users = JSON.parse(localStorage.getItem('users'));
+      if (!users) {
+        notificationConfig.message =
+          'Пользователя с таким email не существует, пожалуйста зарегистрируйтесь';
+        notificationsStore.addNotification(notificationConfig);
```

Comment:
You can add a helper function to use notifications more convenient.

import NOTIFICATION_CONFIG
import { store } from 'react-notifications-component';
function showNotification(options){
store({...NOTIFICATION_CONFIG, ...options})
}
Then you'll be able to use it like this:
import showNotification
showNotification({message: yourMessage, type: 'warning'})

Reply:
done, added a helper function